### PR TITLE
PyLondinium 2019

### LINKED ISSUE #838: PyLondinium 2019
```yaml
- title: PyLondinium 2019
  dir: pylondinium-2019
  youtube_list:
    - https://www.youtube.com/playlist?list=PLWsDEQH0HiPalyM87r_HFIiD53n2k_LMZ
  related_urls:  
    - label: Conference schedule
      url: https://pylondinium.org/schedule.html
  language: eng
  dates:
    begin: 2019-06-14
    end: 2019-06-16
    default: 2019-06-14
  issue: 838
  minimal_download: True
  tags: []
```

### PR DESCRIPTION
**Pull request by akielbowicz**
Fixes #838

### CODE REVIEW

File: pydata-london-2019/videos/bertil-hatt-joao-martins-ab-testing-by-cluster-pydata-london-2019.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  "recorded": "2019-07-13",
+  "related_urls": [
+    {
+      "label": "Conference schedule",
+      "url": "https://pydata.org/london2019/schedule/"
+    }
+  ],
+  "speakers": [
+    "Bertil Hatt",
+    " Jo\u00e3o Martins"
```
Comment (di) [inline pydata-london-2019/videos/bertil-hatt-joao-martins-ab-testing-by-cluster-pydata-london-2019.json]:
```suggestion
    "Jo\u00e3o Martins"
```

File: pydata-london-2019/videos/christopher-samiullah-soledad-galli-testing-and-validating-mlms-pydata-london-2019.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  "recorded": "2019-07-13",
+  "related_urls": [
+    {
+      "label": "Conference schedule",
+      "url": "https://pydata.org/london2019/schedule/"
+    }
+  ],
+  "speakers": [
+    "Christopher Samiullah",
+    " Soledad Galli"
```
Comment (di) [inline pydata-london-2019/videos/christopher-samiullah-soledad-galli-testing-and-validating-mlms-pydata-london-2019.json]:
```suggestion
    "Soledad Galli"
```

File: pydata-london-2019/videos/day-1-lightning-talks-pydata-london-2019.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  "language": "eng",
+  "recorded": "2019-07-13",
+  "related_urls": [
+    {
+      "label": "Conference schedule",
+      "url": "https://pydata.org/london2019/schedule/"
+    }
+  ],
+  "speakers": [
+    "Various Speakers"
```
Comment (di) [inline pydata-london-2019/videos/day-1-lightning-talks-pydata-london-2019.json]: This should just be removed.

File: pydata-london-2019/videos/day-2-lightning-talks-pydata-london-2019.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  "language": "eng",
+  "recorded": "2019-07-13",
+  "related_urls": [
+    {
+      "label": "Conference schedule",
+      "url": "https://pydata.org/london2019/schedule/"
+    }
+  ],
+  "speakers": [
+    "Various Speakers"
```
Comment (di) [inline pydata-london-2019/videos/day-2-lightning-talks-pydata-london-2019.json]: This should just be removed.

File: pydata-london-2019/videos/irene-iriarte-carretero-data-science-for-dinner-recommending-recipes-pydata-london-2019.json [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+  "language": "eng",
+  "recorded": "2019-07-13",
+  "related_urls": [
+    {
+      "label": "Conference schedule",
+      "url": "https://pydata.org/london2019/schedule/"
+    }
+  ],
+  "speakers": [
+    "Irene Iriarte Carretero: Data Science for Dinner"
```
Comment (di) [inline pydata-london-2019/videos/irene-iriarte-carretero-data-science-for-dinner-recommending-recipes-pydata-london-2019.json]:
```suggestion
    "Irene Iriarte Carretero"
```

File: pydata-london-2019/videos/jeffrey-hsu-susannah-klanecek-a-deep-dive-into-nlp-with-pytorch-pydata-london-2019.json [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+  "recorded": "2019-07-13",
+  "related_urls": [
+    {
+      "label": "Conference schedule",
+      "url": "https://pydata.org/london2019/schedule/"
+    }
+  ],
+  "speakers": [
+    "Jeffrey Hsu",
+    " Susannah Klanecek"
```
Comment (di) [inline pydata-london-2019/videos/jeffrey-hsu-susannah-klanecek-a-deep-dive-into-nlp-with-pytorch-pydata-london-2019.json]:
```suggestion
    "Susannah Klanecek"
```

File: pydata-london-2019/videos/maciej-arciuch-karol-grzegorczyk-embeddings-embeddings-everywhere-pydata-london-2019.json [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+  "recorded": "2019-07-13",
+  "related_urls": [
+    {
+      "label": "Conference schedule",
+      "url": "https://pydata.org/london2019/schedule/"
+    }
+  ],
+  "speakers": [
+    "Maciej Arciuch",
+    " Karol Grzegorczyk"
```
Comment (di) [inline pydata-london-2019/videos/maciej-arciuch-karol-grzegorczyk-embeddings-embeddings-everywhere-pydata-london-2019.json]:
```suggestion
    "Karol Grzegorczyk"
```

File: tools/parse_for_detail.py [hunk 8]
```diff
@@ -12,11 +12,16 @@
...
+PATTERNS = {'one' : r'\s*(?P<speakers>.*)\s*:\s*(?P<title>.*)\s*\|\s*(?P<event>.*)',
+            'two' : r'\s*(?P<speakers>.*)\s*-\s*(?P<title>.*)\s*-\s*(?P<event>.*)' }
 
-PAT = re.compile(r'^(.*)')
-SOURCE_KEY = 'description'
-CATEGORY = 'writethedocs-na-2016'
+PYVIDEO_DATA = Path(os.environ['PYVIDEO_DATA'])
+PAT = { key : re.compile(pattern) for key, pattern in PATTERNS.items() }
+SOURCE_KEY = 'title'
+CATEGORY = 'pydata-london-2019'
```
Comment (di) [inline tools/parse_for_detail.py]: Can you put the changes in this file in a separate PR, or revert them? Seems mostly specific to this event.

File: pydata-london-2019/videos/ajay-thampi-interpretable-ai-or-how-i-learned-to-stop-worrying-and-trust-ai-pydata-london-2019.json [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+      "label": "Conference schedule",
+      "url": "https://pydata.org/london2019/schedule/"
+    }
+  ],
+  "speakers": [
+    "Ajay Thampi"
+  ],
+  "tags": [],
+  "thumbnail_url": "https://i.ytimg.com/vi/RqH4KkXPlpk/maxresdefault.jpg",
+  "title": "Interpretable AI or How I Learned to Stop Worrying and Trust AI ",
```
Comment (di) [inline pydata-london-2019/videos/ajay-thampi-interpretable-ai-or-how-i-learned-to-stop-worrying-and-trust-ai-pydata-london-2019.json]:
```suggestion
  "title": "Interpretable AI or How I Learned to Stop Worrying and Trust AI",
```

File: pydata-london-2019/videos/alejandro-saucedo-guide-towards-algorithm-explainability-in-machine-learning-pydata-london-2019.json [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+      "label": "Conference schedule",
+      "url": "https://pydata.org/london2019/schedule/"
+    }
+  ],
+  "speakers": [
+    "Alejandro Saucedo"
+  ],
+  "tags": [],
+  "thumbnail_url": "https://i.ytimg.com/vi/vq8mDiDODhc/maxresdefault.jpg",
+  "title": "Guide towards algorithm explainability in machine learning ",
```
Comment (di) [inline pydata-london-2019/videos/alejandro-saucedo-guide-towards-algorithm-explainability-in-machine-learning-pydata-london-2019.json]:
```suggestion
  "title": "Guide towards algorithm explainability in machine learning",
```

File: pydata-london-2019/videos/anna-schneider-just-ask-designing-intent-driven-algos-pydata-london-2019.json [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+      "label": "Conference schedule",
+      "url": "https://pydata.org/london2019/schedule/"
+    }
+  ],
+  "speakers": [
+    "Anna Schneider"
+  ],
+  "tags": [],
+  "thumbnail_url": "https://i.ytimg.com/vi/6I7qKMMBD2U/maxresdefault.jpg",
+  "title": "Just ask: designing intent-driven algos ",
```
Comment (di) [inline pydata-london-2019/videos/anna-schneider-just-ask-designing-intent-driven-algos-pydata-london-2019.json]:
```suggestion
  "title": "Just ask: designing intent-driven algos",
```

File: pydata-london-2019/videos/anna-veronika-dorogush-mastering-gradient-boosting-with-catboost-pydata-london-2019.json [hunk 12]
```diff
@@ -0,0 +1,25 @@
...
+      "label": "Conference schedule",
+      "url": "https://pydata.org/london2019/schedule/"
+    }
+  ],
+  "speakers": [
+    "Anna Veronika Dorogush"
+  ],
+  "tags": [],
+  "thumbnail_url": "https://i.ytimg.com/vi/usdEWSDisS0/maxresdefault.jpg",
+  "title": "Mastering gradient boosting with CatBoost ",
```
Comment (di) [inline pydata-london-2019/videos/anna-veronika-dorogush-mastering-gradient-boosting-with-catboost-pydata-london-2019.json]:
```suggestion
  "title": "Mastering gradient boosting with CatBoost",
```

File: pydata-london-2019/videos/bertil-hatt-joao-martins-ab-testing-by-cluster-pydata-london-2019.json [hunk 13]
```diff
@@ -0,0 +1,26 @@
...
+  ],
+  "tags": [],
+  "thumbnail_url": "https://i.ytimg.com/vi/KML_EpWLMUM/maxresdefault.jpg",
+  "title": "AB-testing by cluster ",
```
Comment (di) [inline pydata-london-2019/videos/bertil-hatt-joao-martins-ab-testing-by-cluster-pydata-london-2019.json]:
```suggestion
  "title": "AB-testing by cluster",
```

File: pydata-london-2019/videos/cheuk-ting-ho-3-things-about-chatbot-you-don-t-have-to-learn-it-the-hard-way-pydata-london-2019.json [hunk 14]
```diff
@@ -0,0 +1,25 @@
...
+      "label": "Conference schedule",
+      "url": "https://pydata.org/london2019/schedule/"
+    }
+  ],
+  "speakers": [
+    "Cheuk Ting Ho"
+  ],
+  "tags": [],
+  "thumbnail_url": "https://i.ytimg.com/vi/wS5PaQ91wes/maxresdefault.jpg",
+  "title": "3 things about Chatbot (you don't have to learn it the hard way) ",
```
Comment (di) [inline pydata-london-2019/videos/cheuk-ting-ho-3-things-about-chatbot-you-don-t-have-to-learn-it-the-hard-way-pydata-london-2019.json]:
```suggestion
  "title": "3 things about Chatbot (you don't have to learn it the hard way)",
```

File: pydata-london-2019/videos/chie-hayashida-understanding-of-distributed-processing-in-python-pydata-london-2019.json [hunk 15]
```diff
@@ -0,0 +1,25 @@
...
+      "label": "Conference schedule",
+      "url": "https://pydata.org/london2019/schedule/"
+    }
+  ],
+  "speakers": [
+    "Chie Hayashida"
+  ],
+  "tags": [],
+  "thumbnail_url": "https://i.ytimg.com/vi/HoPx3q-LSb8/maxresdefault.jpg",
+  "title": "Understanding of distributed processing in Python ",
```
Comment (di) [inline pydata-london-2019/videos/chie-hayashida-understanding-of-distributed-processing-in-python-pydata-london-2019.json]:
```suggestion
  "title": "Understanding of distributed processing in Python",
```

File: pydata-london-2019/videos/dr-egor-kraev-choose-the-right-neural-generative-model-for-your-problem-pydata-london-2019.json [hunk 16]
```diff
@@ -0,0 +1,25 @@
...
+      "label": "Conference schedule",
+      "url": "https://pydata.org/london2019/schedule/"
+    }
+  ],
+  "speakers": [
+    "Dr. Egor Kraev"
+  ],
+  "tags": [],
+  "thumbnail_url": "https://i.ytimg.com/vi/360bmOagGUQ/maxresdefault.jpg",
+  "title": "Choose the right neural generative model for your problem ",
```
Comment (di) [inline pydata-london-2019/videos/dr-egor-kraev-choose-the-right-neural-generative-model-for-your-problem-pydata-london-2019.json]:
```suggestion
  "title": "Choose the right neural generative model for your problem",
```

File: pydata-london-2019/videos/elena-sharova-how-to-validate-you-client-churn-model-pydata-london-2019.json [hunk 17]
```diff
@@ -0,0 +1,25 @@
...
+      "label": "Conference schedule",
+      "url": "https://pydata.org/london2019/schedule/"
+    }
+  ],
+  "speakers": [
+    "Elena Sharova"
+  ],
+  "tags": [],
+  "thumbnail_url": "https://i.ytimg.com/vi/nRoxiHzL7HY/maxresdefault.jpg",
+  "title": "How to Validate you Client Churn Model ",
```
Comment (di) [inline pydata-london-2019/videos/elena-sharova-how-to-validate-you-client-churn-model-pydata-london-2019.json]:
```suggestion
  "title": "How to Validate you Client Churn Model",
```

File: pydata-london-2019/videos/elina-naydenova-bridging-health-inequalities-through-machine-learning-pydata-london-2019.json [hunk 18]
```diff
@@ -0,0 +1,27 @@
...
+    }
+  ],
+  "speakers": [
+    "Elina Naydenova"
+  ],
+  "tags": [
+    "keynote"
+  ],
+  "thumbnail_url": "https://i.ytimg.com/vi/99lbfZMwvRI/maxresdefault.jpg",
+  "title": "Bridging health inequalities through machine learning ",
```
Comment (di) [inline pydata-london-2019/videos/elina-naydenova-bridging-health-inequalities-through-machine-learning-pydata-london-2019.json]:
```suggestion
  "title": "Bridging health inequalities through machine learning",
```

File: pydata-london-2019/videos/george-cushen-knowledge-graphs-enter-the-hype-cycle-pydata-london-2019.json [hunk 19]
```diff
@@ -0,0 +1,25 @@
...
+      "label": "Conference schedule",
+      "url": "https://pydata.org/london2019/schedule/"
+    }
+  ],
+  "speakers": [
+    "George Cushen"
+  ],
+  "tags": [],
+  "thumbnail_url": "https://i.ytimg.com/vi/P270jIKcEGw/maxresdefault.jpg",
+  "title": "Knowledge graphs --enter- the Hype Cycle ",
```
Comment (di) [inline pydata-london-2019/videos/george-cushen-knowledge-graphs-enter-the-hype-cycle-pydata-london-2019.json]:
```suggestion
  "title": "Knowledge graphs --enter--> the Hype Cycle",
```

File: pydata-london-2019/videos/gianluca-campanella-the-unreasonable-effectiveness-of-feature-hashing-pydata-london-2019.json [hunk 20]
```diff
@@ -0,0 +1,25 @@
...
+      "label": "Conference schedule",
+      "url": "https://pydata.org/london2019/schedule/"
+    }
+  ],
+  "speakers": [
+    "Gianluca Campanella"
+  ],
+  "tags": [],
+  "thumbnail_url": "https://i.ytimg.com/vi/XelrzDtEnPY/maxresdefault.jpg",
+  "title": "The unreasonable effectiveness of feature hashing ",
```
Comment (di) [inline pydata-london-2019/videos/gianluca-campanella-the-unreasonable-effectiveness-of-feature-hashing-pydata-london-2019.json]:
```suggestion
  "title": "The unreasonable effectiveness of feature hashing",
```

File: pydata-london-2019/videos/harrison-pim-image-search-without-image-captions-pydata-london-2019.json [hunk 21]
```diff
@@ -0,0 +1,25 @@
...
+      "label": "Conference schedule",
+      "url": "https://pydata.org/london2019/schedule/"
+    }
+  ],
+  "speakers": [
+    "Harrison Pim"
+  ],
+  "tags": [],
+  "thumbnail_url": "https://i.ytimg.com/vi/DPEap9Rvovk/maxresdefault.jpg",
+  "title": "Image search without image captions ",
```
Comment (di) [inline pydata-london-2019/videos/harrison-pim-image-search-without-image-captions-pydata-london-2019.json]:
```suggestion
  "title": "Image search without image captions",
```

File: pydata-london-2019/videos/igor-gotlibovych-deep-learning-and-time-series-forecasting-for-smarter-energy-pydata-london-2019.json [hunk 22]
```diff
@@ -0,0 +1,25 @@
...
+      "label": "Conference schedule",
+      "url": "https://pydata.org/london2019/schedule/"
+    }
+  ],
+  "speakers": [
+    "Igor Gotlibovych"
+  ],
+  "tags": [],
+  "thumbnail_url": "https://i.ytimg.com/vi/p6mKFs6HVlg/maxresdefault.jpg",
+  "title": "Deep Learning and Time Series Forecasting for Smarter Energy ",
```
Comment (di) [inline pydata-london-2019/videos/igor-gotlibovych-deep-learning-and-time-series-forecasting-for-smarter-energy-pydata-london-2019.json]:
```suggestion
  "title": "Deep Learning and Time Series Forecasting for Smarter Energy",
```

File: pydata-london-2019/videos/imran-rashid-training-intelligent-game-agents-using-deep-reinforcement-learning-pydata-london-2019.json [hunk 23]
```diff
@@ -0,0 +1,25 @@
...
+      "label": "Conference schedule",
+      "url": "https://pydata.org/london2019/schedule/"
+    }
+  ],
+  "speakers": [
+    "Imran Rashid"
+  ],
+  "tags": [],
+  "thumbnail_url": "https://i.ytimg.com/vi/_Hojp_fMOjI/maxresdefault.jpg",
+  "title": "Training intelligent game agents using deep reinforcement learning ",
```
Comment (di) [inline pydata-london-2019/videos/imran-rashid-training-intelligent-game-agents-using-deep-reinforcement-learning-pydata-london-2019.json]:
```suggestion
  "title": "Training intelligent game agents using deep reinforcement learning",
```

File: pydata-london-2019/videos/jan-freyberg-active-learning-in-the-interactive-python-environment-pydata-london-2019.json [hunk 24]
```diff
@@ -0,0 +1,25 @@
...
+      "label": "Conference schedule",
+      "url": "https://pydata.org/london2019/schedule/"
+    }
+  ],
+  "speakers": [
+    "Jan Freyberg"
+  ],
+  "tags": [],
+  "thumbnail_url": "https://i.ytimg.com/vi/W2bJH0iXTKc/maxresdefault.jpg",
+  "title": "Active learning in the interactive python environment ",
```
Comment (di) [inline pydata-london-2019/videos/jan-freyberg-active-learning-in-the-interactive-python-environment-pydata-london-2019.json]:
```suggestion
  "title": "Active learning in the interactive python environment",
```

File: pydata-london-2019/videos/jeffrey-hsu-susannah-klanecek-a-deep-dive-into-nlp-with-pytorch-pydata-london-2019.json [hunk 25]
```diff
@@ -0,0 +1,26 @@
...
+  ],
+  "tags": [],
+  "thumbnail_url": "https://i.ytimg.com/vi/Apx_1erbQB4/maxresdefault.jpg",
+  "title": "A Deep Dive into NLP with PyTorch ",
```
Comment (di) [inline pydata-london-2019/videos/jeffrey-hsu-susannah-klanecek-a-deep-dive-into-nlp-with-pytorch-pydata-london-2019.json]:
```suggestion
  "title": "A Deep Dive into NLP with PyTorch",
```

File: pydata-london-2019/videos/jon-tutcher-data-science-frameworks-managed-services-pydata-london-2019.json [hunk 26]
```diff
@@ -0,0 +1,25 @@
...
+      "label": "Conference schedule",
+      "url": "https://pydata.org/london2019/schedule/"
+    }
+  ],
+  "speakers": [
+    "Jon Tutcher"
+  ],
+  "tags": [],
+  "thumbnail_url": "https://i.ytimg.com/vi/IZ8CBwP5nlA/maxresdefault.jpg",
+  "title": "Data Science Frameworks + Managed Services ",
```
Comment (di) [inline pydata-london-2019/videos/jon-tutcher-data-science-frameworks-managed-services-pydata-london-2019.json]:
```suggestion
  "title": "Data Science Frameworks + Managed Services",
```

File: pydata-london-2019/videos/jonny-brooks-bartlett-how-deliveroo-improved-the-ranking-of-restaurants-pydata-london-2019.json [hunk 27]
```diff
@@ -0,0 +1,25 @@
...
+      "label": "Conference schedule",
+      "url": "https://pydata.org/london2019/schedule/"
+    }
+  ],
+  "speakers": [
+    "Jonny Brooks-Bartlett"
+  ],
+  "tags": [],
+  "thumbnail_url": "https://i.ytimg.com/vi/bG95RmVOn0E/maxresdefault.jpg",
+  "title": "How Deliveroo improved the ranking of restaurants ",
```
Comment (di) [inline pydata-london-2019/videos/jonny-brooks-bartlett-how-deliveroo-improved-the-ranking-of-restaurants-pydata-london-2019.json]:
```suggestion
  "title": "How Deliveroo improved the ranking of restaurants",
```

File: pydata-london-2019/videos/kevin-lemagnen-maintainable-code-in-data-science-pydata-london-2019.json [hunk 28]
```diff
@@ -0,0 +1,25 @@
...
+      "label": "Conference schedule",
+      "url": "https://pydata.org/london2019/schedule/"
+    }
+  ],
+  "speakers": [
+    "Kevin Lemagnen"
+  ],
+  "tags": [],
+  "thumbnail_url": "https://i.ytimg.com/vi/_UE8fsVuQYc/maxresdefault.jpg",
+  "title": "Maintainable code in data science ",
```
Comment (di) [inline pydata-london-2019/videos/kevin-lemagnen-maintainable-code-in-data-science-pydata-london-2019.json]:
```suggestion
  "title": "Maintainable code in data science",
```

File: pydata-london-2019/videos/kirstie-whitaker-the-turing-way-a-how-to-guide-for-reproducible-research-pydata-london-2019.json [hunk 29]
```diff
@@ -0,0 +1,27 @@
...
+    }
+  ],
+  "speakers": [
+    "Kirstie Whitaker"
+  ],
+  "tags": [
+    "keynote"
+  ],
+  "thumbnail_url": "https://i.ytimg.com/vi/IG3PcZ6EhiU/maxresdefault.jpg",
+  "title": "The Turing Way: A how to guide for reproducible research ",
```
Comment (di) [inline pydata-london-2019/videos/kirstie-whitaker-the-turing-way-a-how-to-guide-for-reproducible-research-pydata-london-2019.json]:
```suggestion
  "title": "The Turing Way: A how to guide for reproducible research",
```

File: pydata-london-2019/videos/liam-kirwin-how-am-i-going-to-deal-with-all-of-these-cats-pydata-london-2019.json [hunk 30]
```diff
@@ -0,0 +1,25 @@
...
+      "label": "Conference schedule",
+      "url": "https://pydata.org/london2019/schedule/"
+    }
+  ],
+  "speakers": [
+    "Liam Kirwin"
+  ],
+  "tags": [],
+  "thumbnail_url": "https://i.ytimg.com/vi/hqYQ8Yj9vB0/maxresdefault.jpg",
+  "title": "How am I going to deal with all of these cats ",
```
Comment (di) [inline pydata-london-2019/videos/liam-kirwin-how-am-i-going-to-deal-with-all-of-these-cats-pydata-london-2019.json]:
```suggestion
  "title": "How am I going to deal with all of these cats",
```

File: pydata-london-2019/videos/lorena-a-barba-the-frustration-of-diversity-efforts-in-stem-pydata-london-2019.json [hunk 31]
```diff
@@ -0,0 +1,27 @@
...
+    }
+  ],
+  "speakers": [
+    "Lorena A. Barba"
+  ],
+  "tags": [
+    "keynote"
+  ],
+  "thumbnail_url": "https://i.ytimg.com/vi/THf8_A-RK38/maxresdefault.jpg",
+  "title": "The frustration of diversity efforts in STEM ",
```
Comment (di) [inline pydata-london-2019/videos/lorena-a-barba-the-frustration-of-diversity-efforts-in-stem-pydata-london-2019.json]:
```suggestion
  "title": "The frustration of diversity efforts in STEM",
```

File: pydata-london-2019/videos/lynn-cherny-goofing-off-with-fun-big-data-pydata-london-2019.json [hunk 32]
```diff
@@ -0,0 +1,27 @@
...
+    }
+  ],
+  "speakers": [
+    "Lynn Cherny"
+  ],
+  "tags": [
+    "keynote"
+  ],
+  "thumbnail_url": "https://i.ytimg.com/vi/UzGZtgu3PBM/maxresdefault.jpg",
+  "title": "Goofing Off with Fun (Big) Data ",
```
Comment (di) [inline pydata-london-2019/videos/lynn-cherny-goofing-off-with-fun-big-data-pydata-london-2019.json]:
```suggestion
  "title": "Goofing Off with Fun (Big) Data",
```

File: pydata-london-2019/videos/mahan-hosseinzadeh-prophet-at-scale-to-tune-forecast-time-series-at-spotify-pydata-london-2019.json [hunk 33]
```diff
@@ -0,0 +1,25 @@
...
+      "label": "Conference schedule",
+      "url": "https://pydata.org/london2019/schedule/"
+    }
+  ],
+  "speakers": [
+    "Mahan Hosseinzadeh"
+  ],
+  "tags": [],
+  "thumbnail_url": "https://i.ytimg.com/vi/fegS34ItKcI/maxresdefault.jpg",
+  "title": "Prophet at scale to tune & forecast time series at Spotify ",
```
Comment (di) [inline pydata-london-2019/videos/mahan-hosseinzadeh-prophet-at-scale-to-tune-forecast-time-series-at-spotify-pydata-london-2019.json]:
```suggestion
  "title": "Prophet at scale to tune & forecast time series at Spotify",
```

File: pydata-london-2019/videos/mario-bonamigo-data-science-and-internet-of-things-on-the-edge-pydata-london-2019.json [hunk 34]
```diff
@@ -0,0 +1,25 @@
...
+      "label": "Conference schedule",
+      "url": "https://pydata.org/london2019/schedule/"
+    }
+  ],
+  "speakers": [
+    "Mario Bonamigo"
+  ],
+  "tags": [],
+  "thumbnail_url": "https://i.ytimg.com/vi/HQPi3e8KybM/maxresdefault.jpg",
+  "title": "Data science and Internet of Things on the edge ",
```
Comment (di) [inline pydata-london-2019/videos/mario-bonamigo-data-science-and-internet-of-things-on-the-edge-pydata-london-2019.json]:
```suggestion
  "title": "Data science and Internet of Things on the edge",
```

File: pydata-london-2019/videos/mark-farragher-on-the-path-to-causal-inference-pydata-london-2019.json [hunk 35]
```diff
@@ -0,0 +1,25 @@
...
+      "label": "Conference schedule",
+      "url": "https://pydata.org/london2019/schedule/"
+    }
+  ],
+  "speakers": [
+    "Mark Farragher"
+  ],
+  "tags": [],
+  "thumbnail_url": "https://i.ytimg.com/vi/vg9TZa-AgC8/maxresdefault.jpg",
+  "title": "On the Path to Causal Inference ",
```
Comment (di) [inline pydata-london-2019/videos/mark-farragher-on-the-path-to-causal-inference-pydata-london-2019.json]:
```suggestion
  "title": "On the Path to Causal Inference",
```

File: pydata-london-2019/videos/michael-sugimura-computer-vision-and-nlp-for-multi-task-fashion-attrb-modeling-pydata-london-2019.json [hunk 36]
```diff
@@ -0,0 +1,25 @@
...
+      "label": "Conference schedule",
+      "url": "https://pydata.org/london2019/schedule/"
+    }
+  ],
+  "speakers": [
+    "Michael Sugimura"
+  ],
+  "tags": [],
+  "thumbnail_url": "https://i.ytimg.com/vi/YGD5X0nvQQA/maxresdefault.jpg",
+  "title": "Computer Vision and NLP for Multi-Task Fashion Attrb. Modeling ",
```
Comment (di) [inline pydata-london-2019/videos/michael-sugimura-computer-vision-and-nlp-for-multi-task-fashion-attrb-modeling-pydata-london-2019.json]:
```suggestion
  "title": "Computer Vision and NLP for Multi-Task Fashion Attrb. Modeling",
```

File: pydata-london-2019/videos/michal-mucha-build-and-deploy-an-end-to-end-streaming-nlp-insight-system-pydata-london-2019.json [hunk 37]
```diff
@@ -0,0 +1,25 @@
...
+      "label": "Conference schedule",
+      "url": "https://pydata.org/london2019/schedule/"
+    }
+  ],
+  "speakers": [
+    "Michal Mucha"
+  ],
+  "tags": [],
+  "thumbnail_url": "https://i.ytimg.com/vi/fk_uckAoXJ8/maxresdefault.jpg",
+  "title": "Build and Deploy an End-to-end Streaming NLP Insight System ",
```
Comment (di) [inline pydata-london-2019/videos/michal-mucha-build-and-deploy-an-end-to-end-streaming-nlp-insight-system-pydata-london-2019.json]:
```suggestion
  "title": "Build and Deploy an End-to-end Streaming NLP Insight System",
```

File: pydata-london-2019/videos/omer-yuksel-understanding-exchange-network-dynamics-with-python-pydata-london-2019.json [hunk 38]
```diff
@@ -0,0 +1,25 @@
...
+      "label": "Conference schedule",
+      "url": "https://pydata.org/london2019/schedule/"
+    }
+  ],
+  "speakers": [
+    "Omer Yuksel"
+  ],
+  "tags": [],
+  "thumbnail_url": "https://i.ytimg.com/vi/kvQ5WdJ49CQ/maxresdefault.jpg",
+  "title": "Understanding exchange network dynamics with Python ",
```
Comment (di) [inline pydata-london-2019/videos/omer-yuksel-understanding-exchange-network-dynamics-with-python-pydata-london-2019.json]:
```suggestion
  "title": "Understanding exchange network dynamics with Python",
```

File: pydata-london-2019/videos/raoul-gabriel-urma-kevin-lemagnen-adv-software-testing-for-data-scientists-pydata-london-2019.json [hunk 39]
```diff
@@ -0,0 +1,26 @@
...
+      "url": "https://pydata.org/london2019/schedule/"
+    }
+  ],
+  "speakers": [
+    "Raoul-Gabriel Urma",
+    "Kevin Lemagnen"
+  ],
+  "tags": [],
+  "thumbnail_url": "https://i.ytimg.com/vi/WTj6T0QdHHM/maxresdefault.jpg",
+  "title": "Adv. Software Testing for Data Scientists ",
```
Comment (di) [inline pydata-london-2019/videos/raoul-gabriel-urma-kevin-lemagnen-adv-software-testing-for-data-scientists-pydata-london-2019.json]:
```suggestion
  "title": "Adv. Software Testing for Data Scientists",
```

File: pydata-london-2019/videos/ruben-van-de-geer-a-primer-or-refresher-on-linear-algebra-for-data-science-pydata-london-2019.json [hunk 40]
```diff
@@ -0,0 +1,25 @@
...
+      "label": "Conference schedule",
+      "url": "https://pydata.org/london2019/schedule/"
+    }
+  ],
+  "speakers": [
+    "Ruben van de Geer"
+  ],
+  "tags": [],
+  "thumbnail_url": "https://i.ytimg.com/vi/Qz58vTa8-SY/maxresdefault.jpg",
+  "title": "A Primer (or Refresher) On Linear Algebra for Data Science ",
```
Comment (di) [inline pydata-london-2019/videos/ruben-van-de-geer-a-primer-or-refresher-on-linear-algebra-for-data-science-pydata-london-2019.json]:
```suggestion
  "title": "A Primer (or Refresher) On Linear Algebra for Data Science",
```

File: pydata-london-2019/videos/trevor-sidery-guillermo-barquero-productionising-data-science-at-scale-pydata-london-2019.json [hunk 41]
```diff
@@ -0,0 +1,26 @@
...
+      "url": "https://pydata.org/london2019/schedule/"
+    }
+  ],
+  "speakers": [
+    "Trevor Sidery",
+    "Guillermo Barquero"
+  ],
+  "tags": [],
+  "thumbnail_url": "https://i.ytimg.com/vi/gHTJ9PAnLQ0/maxresdefault.jpg",
+  "title": "Productionising Data Science at Scale ",
```
Comment (di) [inline pydata-london-2019/videos/trevor-sidery-guillermo-barquero-productionising-data-science-at-scale-pydata-london-2019.json]:
```suggestion
  "title": "Productionising Data Science at Scale",
```

File: pydata-london-2019/videos/vincent-warmerdam-how-to-constrain-artificial-stupidity-pydata-london-2019.json [hunk 42]
```diff
@@ -0,0 +1,25 @@
...
+      "label": "Conference schedule",
+      "url": "https://pydata.org/london2019/schedule/"
+    }
+  ],
+  "speakers": [
+    "Vincent Warmerdam"
+  ],
+  "tags": [],
+  "thumbnail_url": "https://i.ytimg.com/vi/Z8MEFI7ZJlA/maxresdefault.jpg",
+  "title": "How to Constrain Artificial Stupidity ",
```
Comment (di) [inline pydata-london-2019/videos/vincent-warmerdam-how-to-constrain-artificial-stupidity-pydata-london-2019.json]:
```suggestion
  "title": "How to Constrain Artificial Stupidity",
```

File: pydata-london-2019/videos/zack-akil-safer-cycling-with-an-edge-tpu-watching-your-back-pydata-london-2019.json [hunk 43]
```diff
@@ -0,0 +1,25 @@
...
+      "label": "Conference schedule",
+      "url": "https://pydata.org/london2019/schedule/"
+    }
+  ],
+  "speakers": [
+    "Zack Akil"
+  ],
+  "tags": [],
+  "thumbnail_url": "https://i.ytimg.com/vi/UmDeKDgFRj4/maxresdefault.jpg",
+  "title": "Safer Cycling with an Edge TPU watching your back ",
```
Comment (di) [inline pydata-london-2019/videos/zack-akil-safer-cycling-with-an-edge-tpu-watching-your-back-pydata-london-2019.json]:
```suggestion
  "title": "Safer Cycling with an Edge TPU watching your back",
```

**Comment by Daniel-at-github** [comment]
Thanks for your work @akielbowicz !!!

Somehow pydata-london-2019 commit got mixed here. Comments from @di should be solved in #850.

I continued your work adding descriptions to it, and tried this with no luck:
> Add more commits by pushing to the pylondinium-2019 branch on akielbowicz/data.

Solved it somehow opening #851 and push your and my changes there.
I will close this and continue it in #851

**Pull request closed without merging**